entrypoint.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
```

Comment:
Why not the entrypoint file of the parent image?

Reply:
I found a bug and testing the fix with this PR.